x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -239,8 +239,11 @@ export class ExecutionHandler<
           logActions.push({
             id: action.id,
             typeId: action.actionTypeId,
-            alertId: 'summary',
-            alertGroup: action.group,
+            alertSummary: {
```

Comment:
seems like our unit tests for summary actions don't test what `logActions` is called with. may be worth updating if simple to

Reply:
Resolved in this commit [4889843](https://github.com/elastic/kibana/pull/150088/commits/4889843abf7ace661c036545ebd50b723872ed5c)